pytorch_lightning/core/lightning.py

```diff
@@ -1473,6 +1478,64 @@ def untoggle_optimizer(self, optimizer_idx: int):
         # save memory
         self._param_requires_grad_state = {}
 
+    def clip_gradients(self, optimizer: Optimizer, gradient_clip_val: Union[int, float], gradient_clip_algorithm: str):
```

Comment:
Can you scan the docs for a place where to advertise this hook?

Also add it to https://pytorch-lightning.readthedocs.io/en/latest/common/lightning_module.html#hooks